### DISCUSSION TITLE
Busco equipo

### DISCUSSION THREAD

**Discussion by AliciaGaona** [Launch-X-Latam/MisionFrontEnd#515] [Team Posts]
Buenas tardes busco equipo, soy FrontEnd  ^^, no hajia podido escribir y veo que ya hay varios equipos formados. Si a alguien le falta un integrante o no tiene equipo.

**Comment by FrankAV27** [comment]
Hola mi nombre es Franklin y quisiera unirme a tu grupo mi correo es: lx09414@innovaccion.mx

**Comment by iam-leo** [comment]
Hola soy Leo recién termino de ver el live de ayer, no se muy bien como es esto de los equipos pero si queda cupo me puedo sumar? tengo algunos conocimientos en frontend (básico nada de frameworks aún). Mi correo es: lx13348@innovaccion.mx

**Comment by Saulflores01** [comment]
Hola
Me gustaría ser parte de tu equipo
Dejo mi correo por cualquier cosa lx04135@innovacción.mx

**Comment by AliciaGaona** [comment]
holaa, siii seamos equuipo todos, solo que no se cuantos tengamos que ser o como este permitida la comunicación

**Comment by AleBT75** [comment]
Buenas tardes, aún hay vacantes? :D

**Comment by UlisesNh** [comment]
Hola, aun quepo?

**Comment by FrankAV27** [comment]
> solo que no se cuantos tengamos que ser o como este permitida la comunicación

Si no mal escuche solo son 5  c:

**Comment by FrankAV27** [comment]
@AliciaGaona  Podrías ir creando el grupo en Teams ?

**Comment by iam-leo** [comment]
> solo que no se cuantos tengamos que ser o como este permitida la comunicación
> 
> Si no mal escuche solo son 5 c:

Sí yo también escuché que era hasta 5 pero tampoco está confirmado creo! Supongo que no habrá ningún problema si somos 6! Ahora los sigo a todos en Github y redes si es que las tienen, hasta que nos digan como proceder.

**Comment by PepoGE** [comment]
Hola! Me llamo José Luis y estaría encantado de formar parte del equipo, ando cursando una Ingeniería en TI,  tengo conocimientos en FrontEnd, programación en Java y experiencia con UI, siempre dispuesto a aprender y ayudar con lo que pueda aportar :D 
Mi correo es: lx03228@innovaccion.mx

**Comment by JesrigPineda** [comment]
Hola 🙌
Me gustaría ser parte de tu equipo, si aun hay espacio,
Dejo mi correo por cualquier cosa lx04646@innovaccion.mx 😊

**Comment by LaBalaElmo** [comment]
Buenas, yo también me encuentro buscando grupo por si aun necesitan a uno. Este es mi correo para cualquier cosa: lx10061@innovaccion.mx.
Un saludo.

**Comment by LuisRondon11** [comment]
Buenos días. También me encuentro en busca de equipo, quién más falta por equipo mi correo de innovaccion es lx11044@innovaccion.mx.
Saludos

**Comment by safraZX** [comment]
yo tambien busco equipo lx1227@innovaccion.mx

**Comment by mafer23** [comment]
Porque no SE crea un discord para no estar enviando mensajes.

**Comment by JesrigPineda** [comment]
> Porque no SE crea un discord para no estar enviando mensajes.

Hola mafer23, ya tienes equipo? 😁

**Comment by mafer23** [comment]
No ,Tengo , creamos uno jajaja , variadito .

**Comment by JesrigPineda** [comment]
> 

Va me late 😉

**Comment by LuisRondon11** [comment]
Puedo unirme les, no tengo equipo

**Comment by mafer23** [comment]
De Las niñas quien se quiere unir y de Los niños quien , creamos un discord
para no hacer spam aqui por la Gente necesita despejado El correo.

**Comment by LuisRondon11** [comment]
Mi usuario es hintax#4439

**Comment by iam-leo** [comment]
Con la gente que comentó en este hilo se podrían armar dos grupos de 6. Siguiendo el orden en que fueron comentando, podría ser: 

Grupo-1: @AliciaGaona @FrankAV27 @iam-leo @Saulflores01 @AleBT75 @UlisesNh 

Grupo-2: @PepoGE @JesrigPineda @LaBalaElmo @LuisRondon11 @safraZX @mafer23 

Que les parece? están de acuerdo? Resta saber si se puede hacer de 6 integrantes o es estrictamente de 5, hoy preguntamos en el live. En caso de que haya alguien que ya haya conseguido grupo por favor avise así le damos lugar a alguien mas que quiera sumarse.

**Comment by mafer23** [comment]
https://discord.gg/QZa2UAHu Mira faltan 2 pero, si quieren creamos Los
canales para cada grupo.
